run

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+# Cross-platform script to run commands inside or outside container
+# Automatically detects context and routes commands appropriately
+# Usage: ./run <any-command-and-args>
+
+set -e  # Exit on any error
+
+# Project-specific configuration
+COMPOSE_FILE="docker/compose.yaml"
+SERVICE_NAME="grassroots_dev"
+
+# Get the directory where this script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Function to find the project root
+find_project_root() {
+    local current_dir="$SCRIPT_DIR"
+
+    # Walk up the directory tree looking for docker/compose.yaml
+    while [[ "$current_dir" != "/" ]]; do
+        if [[ -f "$current_dir/$COMPOSE_FILE" ]]; then
+            echo "$current_dir"
+            return 0
+        fi
+        current_dir="$(dirname "$current_dir")"
+    done
+
+    # Check if compose file exists
+    echo "Error: $COMPOSE_FILE not found in project root"
+    echo "Make sure you're running this from within the project directory"
+    exit 1
+}
```

Comment:
The `run` file is in the project root, so `SCRIPT_DIR` will always be the project root

Reply:
done